### DISCUSSION TITLE
Is this an error ? Can I ignore it?

### DISCUSSION THREAD

**Discussion by ozgurerdogan** [psy0rz/zfs_autobackup#163] [Q&A]
From:
zfs-autobackup -v s4d1 D3 --keep-source=5 --keep-target=5 --strip-path=2 --clear-mountpoint --clear-refreservatio --ignore-transfer-errors --no-holds --snapshot-format %Y-%m-%d-%H:%M:%S --destroy-missing=10d

And I see:
NOTE: Source and target are on the same host, excluding target-path from selection.
But all goes fine. 
Btw this is a great tool. Thanks.

**Comment by psy0rz** [comment]
its just a note that it will it ignore the target path. It will only select datasets to backup that are not in the target-path. 

This is what most people want anyway, but its important enough to note. :)

**Comment by ozgurerdogan** [comment]
So target-path is here "D3" ? So I have to add it right ?
